[2025-03-21T08:16:40.011-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-21T08:16:40.019-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: proceso_etl.tarea_datawrangling manual__2025-03-21T13:16:08.356970+00:00 [queued]>
[2025-03-21T08:16:40.024-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: proceso_etl.tarea_datawrangling manual__2025-03-21T13:16:08.356970+00:00 [queued]>
[2025-03-21T08:16:40.024-0500] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-21T08:16:40.040-0500] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): tarea_datawrangling> on 2025-03-21 13:16:08.356970+00:00
[2025-03-21T08:16:40.048-0500] {standard_task_runner.py:72} INFO - Started process 21572 to run task
[2025-03-21T08:16:40.050-0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'proceso_etl', 'tarea_datawrangling', 'manual__2025-03-21T13:16:08.356970+00:00', '--job-id', '56', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl.py', '--cfg-path', '/tmp/tmptu4n_btk']
[2025-03-21T08:16:40.051-0500] {standard_task_runner.py:105} INFO - Job 56: Subtask tarea_datawrangling
[2025-03-21T08:16:40.077-0500] {task_command.py:467} INFO - Running <TaskInstance: proceso_etl.tarea_datawrangling manual__2025-03-21T13:16:08.356970+00:00 [running]> on host luis-GF65
[2025-03-21T08:16:40.131-0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='proceso_etl' AIRFLOW_CTX_TASK_ID='tarea_datawrangling' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T13:16:08.356970+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-21T13:16:08.356970+00:00'
[2025-03-21T08:16:40.132-0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-21T08:16:41.589-0500] {data_wrangling.py:149} INFO - 1. Terminando formateo!!
[2025-03-21T08:16:51.157-0500] {_client.py:1025} INFO - HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
[2025-03-21T08:16:53.723-0500] {_client.py:1025} INFO - HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
[2025-03-21T08:16:53.729-0500] {data_wrangling.py:186} ERROR - Los clasificados como 'venta' no son considerablemente más caros que los de alquiler. Verificar información
[2025-03-21T08:16:53.730-0500] {data_wrangling.py:187} INFO - Los clasificados como 'venta' cuestan 1.0 veces más que los de alquiler
[2025-03-21T08:16:53.731-0500] {data_wrangling.py:199} INFO - 2. Terminando clasificación de propiedades con data inconsistente!!
[2025-03-21T08:17:38.806-0500] {data_wrangling.py:214} INFO - 3. Terminando obtención de dirección y precisión de coordenadas!!
[2025-03-21T08:17:38.809-0500] {logging_mixin.py:190} INFO - 0      None
2      None
3      None
4      None
6      None
       ... 
244    None
245    None
246    None
248    None
249     NaN
Name: precision, Length: 220, dtype: object
[2025-03-21T08:17:38.809-0500] {data_wrangling.py:220} INFO - 220
[2025-03-21T08:17:38.809-0500] {data_wrangling.py:221} INFO - 220
[2025-03-21T08:17:57.001-0500] {_client.py:1025} INFO - HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
[2025-03-21T08:17:57.309-0500] {_client.py:1025} INFO - HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
[2025-03-21T08:18:02.019-0500] {_client.py:1025} INFO - HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
[2025-03-21T08:18:05.195-0500] {_client.py:1025} INFO - HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
[2025-03-21T08:18:16.560-0500] {_client.py:1025} INFO - HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
[2025-03-21T08:18:17.127-0500] {_client.py:1025} INFO - HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
[2025-03-21T08:18:27.032-0500] {_client.py:1025} INFO - HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
[2025-03-21T08:18:28.951-0500] {_client.py:1025} INFO - HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
[2025-03-21T08:18:31.613-0500] {_client.py:1025} INFO - HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
[2025-03-21T08:18:31.638-0500] {logging_mixin.py:190} WARNING - /home/luis/Proyectos/PrecioDeDepartamentos/Precios-depas-git/ProyectoDepas/2.0.StaggingLayer/StagingLayerProject/Pipeline/utils/general.py:23 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-03-21T08:18:31.640-0500] {logging_mixin.py:190} INFO - El % de coincidencias para la columna 'area': 92.38% de 105
[2025-03-21T08:18:31.643-0500] {logging_mixin.py:190} WARNING - /home/luis/Proyectos/PrecioDeDepartamentos/Precios-depas-git/ProyectoDepas/2.0.StaggingLayer/StagingLayerProject/Pipeline/utils/general.py:23 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-03-21T08:18:31.644-0500] {logging_mixin.py:190} INFO - El % de coincidencias para la columna 'dormitorios': 97.93% de 145
[2025-03-21T08:18:31.647-0500] {logging_mixin.py:190} WARNING - /home/luis/Proyectos/PrecioDeDepartamentos/Precios-depas-git/ProyectoDepas/2.0.StaggingLayer/StagingLayerProject/Pipeline/utils/general.py:23 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-03-21T08:18:31.648-0500] {logging_mixin.py:190} INFO - El % de coincidencias para la columna 'banos': 80.00% de 145
[2025-03-21T08:18:31.650-0500] {logging_mixin.py:190} WARNING - /home/luis/Proyectos/PrecioDeDepartamentos/Precios-depas-git/ProyectoDepas/2.0.StaggingLayer/StagingLayerProject/Pipeline/utils/general.py:23 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-03-21T08:18:31.652-0500] {logging_mixin.py:190} INFO - El % de coincidencias para la columna 'antiguedad': 90.24% de 41
[2025-03-21T08:18:31.653-0500] {data_wrangling.py:270} INFO - La variable area redujo 5 nulos
[2025-03-21T08:18:31.654-0500] {data_wrangling.py:270} INFO - La variable dormitorios redujo 12 nulos
[2025-03-21T08:18:31.654-0500] {data_wrangling.py:270} INFO - La variable banos redujo 8 nulos
[2025-03-21T08:18:31.655-0500] {data_wrangling.py:270} INFO - La variable antiguedad redujo 0 nulos
[2025-03-21T08:18:31.657-0500] {data_wrangling.py:279} INFO - 4. Terminando imputación de nulos de variables básicas con la descripción!!
[2025-03-21T08:18:39.601-0500] {_client.py:1025} INFO - HTTP Request: POST https://api.openai.com/v1/chat/completions "HTTP/1.1 200 OK"
[2025-03-21T08:18:40.809-0500] {data_wrangling.py:336} INFO - 5. Terminando imputación de nulos de coordenadas con la descripción!!
[2025-03-21T08:18:43.209-0500] {python.py:240} INFO - Done. Returned value was: None
[2025-03-21T08:18:43.213-0500] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-21T08:18:43.213-0500] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=proceso_etl, task_id=tarea_datawrangling, run_id=manual__2025-03-21T13:16:08.356970+00:00, execution_date=20250321T131608, start_date=20250321T131640, end_date=20250321T131843
[2025-03-21T08:18:43.278-0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-21T08:18:43.292-0500] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-21T08:18:43.303-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
